sdk/documentintelligence/ai-document-intelligence-rest/package.json

```diff
@@ -1,123 +1,95 @@
-    "pack": "npm pack 2>&1",
-    "test:browser": "echo skipped",
-    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
-    "test": "npm run clean && npm run build:test && npm run unit-test",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
-    "unit-test:browser": "echo skipped",
-    "build": "npm run clean && tsc -p . && dev-tool run bundle && mkdirp ./review && dev-tool run extract-api"
-  },
   "sideEffects": false,
   "autoPublish": false,
-  "dependencies": {
-    "@azure/core-auth": "^1.6.0",
-    "@azure-rest/core-client": "^1.2.0",
-    "@azure/core-rest-pipeline": "^1.14.0",
-    "@azure/logger": "^1.0.0",
-    "tslib": "^2.2.0",
-    "@azure/core-paging": "^1.5.0",
-    "@azure/core-lro": "^2.5.4",
-    "@azure/abort-controller": "^1.0.0"
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": ["esm", "commonjs"],
+    "esmDialects": ["browser", "react-native"],
+    "selfLink": false
   },
-  "devDependencies": {
-    "@microsoft/api-extractor": "^7.31.1",
-    "autorest": "latest",
-    "@types/node": "^18.0.0",
-    "dotenv": "^16.0.0",
-    "eslint": "^8.0.0",
-    "mkdirp": "^3.0.1",
-    "rimraf": "^5.0.0",
-    "source-map-support": "^0.5.9",
-    "typescript": "~5.5.3",
-    "@azure/dev-tool": "^1.0.0",
-    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure-tools/test-credential": "^1.0.0",
-    "@azure/identity": "^4.0.1",
-    "@azure-tools/test-recorder": "^3.0.0",
-    "mocha": "^10.0.0",
-    "@types/mocha": "^10.0.0",
-    "cross-env": "^7.0.2",
-    "@types/chai": "^4.2.8",
-    "chai": "^4.2.0",
-    "karma-chrome-launcher": "^3.0.0",
-    "karma-coverage": "^2.0.0",
-    "karma-env-preprocessor": "^0.1.1",
-    "karma-firefox-launcher": "^2.1.2",
-    "karma-junit-reporter": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-mocha": "^2.0.1",
-    "karma-source-map-support": "~1.4.0",
-    "karma-sourcemap-loader": "^0.4.0",
-    "karma": "^6.2.0",
-    "nyc": "^17.0.0",
-    "ts-node": "^10.0.0"
+  "type": "module",
+  "keywords": ["node", "azure", "cloud", "typescript", "browser", "isomorphic"],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "files": ["dist", "README.md", "LICENSE", "review/*"],
+  "sdk-type": "client",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/documentintelligence/ai-document-intelligence-rest/README.md",
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "//metadata": {
     "constantPaths": [
       {
         "path": "src/documentIntelligence.ts",
-        "prefix": "userAgentInfo"
+        "prefix": "package-version"
```

Comment:
is this intended? @joheredi @MaryGao

Reply:
no, this should be a issue, @HarshaNalluru could you manually revert this back?

@kazrael2119 could you help fix this?

Reply:
fixes in latest codegen.